Pg search

chamyndu opened the pull request.

Commit:
fucking ingredients favs

Commit:
ingredients favs, i can like dislike and when clicked stays on the same ingredient

Commit:
solve conflict

Bartducrot commented on app/assets/stylesheets/layout/_utilities.scss:
```diff
@@ -31,6 +31,6 @@ margin-bottom: 90px;
 
-.btn-success {
- margin-top: 10px;
+.row {
```
@chamyndu ".row" is already a bootstrap class, you shouldn't override it here, use an other class name (it could break the bootstrap class)

Bartducrot commented on app/views/users/show.html.erb:
```diff
@@ -1,10 +1,11 @@
-<div class="container user-profile text-center">
+<div style="padding-top: 80px" class="container user-profile text-center">
```
@chamyndu why did you put a style attribute instead of using a class?

Commit:
Create _utilities.scss

Sorry this was a little test I tried, but forgot to remove it after. So it's useless.

Commit:
Update show.html.erb

Also here, forgot to remove my test.

Commit:
Update ingredients.html.erb

adding "main-container" which is added to every page so that the navbar doesn't cover the top of the page

chamyndu reviewed (commented):
I amended 2 things you pointed out and 1 thing I noticed when I was going through the change, adding "main-container" to the ingredients page
